content/en/service_management/on-call/automations.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Automations
+further_reading:
+- link: '/service_management/on-call/'
+  tag: 'Documentation'
+  text: 'Datadog On-Call'
+- link: "https://docs.datadoghq.com/service_management/workflows/"
+  tag: "Documentation"
+  text: "Workflow Automation"
+---
+
+## Overview
+
+Automations allow you to trigger actions based on events in On-Call. There are two types:
+- **Handover automations**: Triggered automatically when an on-call shift changes.
+- **[Workflow automations][1]**: A low-code/no-code solution that lets you define actions based on various on-call activities.
+
+
+## Handover automations
+Handover automations run automatically at the start or end of an on-call shift. They're designed to handle tasks that teams typically manage with custom scripts—such as notifying other systems about who's on-call, updating internal dashboards, or rotating credentials.
+
+By using built-in automation instead of maintaining cron jobs or custom tools, you can streamline operations, eliminate manual steps, and ensure the right actions always run when a shift changes.
```

Comment:
```suggestion
By using built-in automations instead of maintaining cron jobs or custom tools, you can streamline operations, eliminate manual steps, and ensure the right actions always run when a shift changes.
```